Remotion/Web/Development.WebTesting/Accessibility/AccessibilityResultViolationsExtensions.cs

```diff
@@ -33,7 +33,7 @@ public static class AccessibilityResultViolationsExtensions
     /// Removes elements with matching CSS selector from the given violations.
     /// </summary>
     public static IEnumerable<AccessibilityRuleResult> IgnoreByCssSelector (
-        [NotNull] this IEnumerable<AccessibilityRuleResult> violations,
+        [NotNull] this IReadOnlyCollection<AccessibilityRuleResult> violations,
```

Comment:
This is due to https://re-motion.atlassian.net/browse/RM-5035 and required because of the change changing the checked in .Sources files (utilities) to file links.